src/include/mpir_comm.h

```diff
@@ -89,6 +89,8 @@ enum MPIR_COMM_HINT_PREDEFINED_t {
     MPIR_COMM_HINT_EAGER_THRESH,        /* ch3 */
     MPIR_COMM_HINT_EAGAIN,      /* ch4:ofi */
     /* dynamic hints starts here */
+    MPIR_COMM_HINT_VCI_IDX_SENDER,
+    MPIR_COMM_HINT_VCI_IDX_RECEIVER,
```

Comment:
Move above the comment `/* dynamic hints starts here */`.  Both are still "static" hints. Dynamic hints will be tracked by a dynamic variable rather than a constant index.

Reply:
Done